# HG changeset patch
# User Brian Gough <bjg@gnu.org>
# Date 1231824413 18000
#      Tue Jan 13 00:26:53 2009 -0500
# Node ID b88596e8f341c1bb5a14007f2d1b9d2cfc6f2ef7
# Parent  e76b92c7f77984e1cd3862b947a273ee6988eea6
[docs] non singleton => non-singleton

diff --git a/scripts/general/cumtrapz.m b/scripts/general/cumtrapz.m
--- a/scripts/general/cumtrapz.m
+++ b/scripts/general/cumtrapz.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {@var{z} =} cumtrapz (@var{y})
 ## @deftypefnx {Function File} {@var{z} =} cumtrapz (@var{x}, @var{y})
 ## @deftypefnx {Function File} {@var{z} =} cumtrapz (@dots{}, @var{dim})
 ## 
 ## Cumulative numerical integration using trapezoidal method.
 ## @code{cumtrapz (@var{y})} computes the cumulative integral of the 
-## @var{y} along the first non singleton dimension. If the argument 
+## @var{y} along the first non-singleton dimension. If the argument 
 ## @var{x} is omitted a equally spaced vector is assumed. @code{cumtrapz 
 ## (@var{x}, @var{y})} evaluates the cumulative integral with respect 
 ## to @var{x}.
 ##  
 ## @seealso{trapz,cumsum}
 ## @end deftypefn
 
 ## Author:	Kai Habel <kai.habel@gmx.de>
diff --git a/scripts/general/trapz.m b/scripts/general/trapz.m
--- a/scripts/general/trapz.m
+++ b/scripts/general/trapz.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {@var{z} =} trapz (@var{y})
 ## @deftypefnx {Function File} {@var{z} =} trapz (@var{x}, @var{y})
 ## @deftypefnx {Function File} {@var{z} =} trapz (@dots{}, @var{dim})
 ## 
 ## Numerical integration using trapezoidal method. @code{trapz
 ## (@var{y})} computes the integral of the @var{y} along the first
-## non singleton dimension. If the argument @var{x} is omitted a 
+## non-singleton dimension. If the argument @var{x} is omitted a 
 ## equally spaced vector is assumed. @code{trapz (@var{x}, @var{y})} 
 ## evaluates the integral with respect to @var{x}.
 ##  
 ## @seealso{cumtrapz}
 ## @end deftypefn
 
 ## Author:	Kai Habel <kai.habel@gmx.de>
 ##
diff --git a/scripts/statistics/base/iqr.m b/scripts/statistics/base/iqr.m
--- a/scripts/statistics/base/iqr.m
+++ b/scripts/statistics/base/iqr.m
@@ -17,17 +17,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} iqr (@var{x}, @var{dim})
 ## If @var{x} is a vector, return the interquartile range, i.e., the
 ## difference between the upper and lower quartile, of the input data.
 ##
-## If @var{x} is a matrix, do the above for first non singleton
+## If @var{x} is a matrix, do the above for first non-singleton
 ## dimension of @var{x}. If the option @var{dim} argument is given,
 ## then operate along this dimension.
 ## @end deftypefn
 
 ## Author KH <Kurt.Hornik@wu-wien.ac.at>
 ## Description: Interquartile range
 
 function y = iqr (x, dim)
